db/migrate/20170711193959_add_column_to_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToAnswers < ActiveRecord::Migration
+  def change
+    add_column :answers, :accepted, :boolean
```

Comment:
Rather than another migration, you can simply say here:

`add_column :answers, :accepted, :boolean, default: true